### ISSUE TITLE
Next Episode section is missing/skipping the next episode

### ISSUE THREAD

**Issue by Nickbert7** [jellyfin/jellyfin-androidtv#199]
On the episode detail page the next episode is missing/skipped
![20191114_235240](https://user-images.githubusercontent.com/14938497/68903021-1df03300-073a-11ea-839d-ac8bd8ed0ec9.jpg)

It might make sense to also have a dedicated "next episode" button

**Comment by Nickbert7** [comment]
It does not occur on all shows. So far it seems only to happen on seasons/shows with missing episodes or with double-episodes.

**Comment by Nickbert7** [comment]
When I enable the "display missing episodes" setting within the library, the next episode is displayed correct int in the app.
Only for seasons with a double episode the issue is still present.
As an alternative solution, would it be possible to show all episodes of the season on this screen but "hide" the previous episodes left outside the screen. So if you navigate down the next (or current) episode is selected and if you navigate to the left also the previous episodes can be selected.
So we might get rid of the previous button to get a cleaner interface. (Or add the next button to be consistent)

**Comment by Nickbert7** [comment]
The problem still exists in the current version

**Comment by innologi** [comment]
I came here looking if my issue of skipped episodes in "next episodes" was already reported, and upon reading this one, I can now confirm that the issue only occurs for shows I have deleted one or more episodes from. Enabling the missing episode feature also remedies the issue for me.

I would like to add that for me, the number of deleted episodes directly relates to the number of episodes being skipped. For example, I'm looking at:
- a single season, 52-episode show (+ specials shown in season)
- first 26 episodes watched and 7 specials, of which I deleted 20 episodes and 6 specials
- I enter the episode 27 details
- first on next episodes shows: episode 40
- now, if I count the number of episodes+specials in the season that were skipped leading up to episode 40, I count exactly 20, which is the number of actual season episodes deleted.
